network/alsp/manager/manager.go

```diff
@@ -237,6 +259,77 @@ func (m *MisbehaviorReportManager) HandleMisbehaviorReport(channel channels.Chan
 	lg.Debug().Msg("misbehavior report submitted")
 }
 
+// startHeartbeatTicks starts the heartbeat ticks ticker to tick at the given intervals. It is a blocking function, and
+// should be called in a separate goroutine. It returns when the context is canceled. Hearbeats are recurring events that
+// are used to perform periodic tasks.
+// Args:
+//
+//	ctx: the context.
+//	interval: the interval between two ticks.
+//
+// Returns:
+//
+//	none.
+func (m *MisbehaviorReportManager) startHeartbeatTicks(ctx irrecoverable.SignalerContext, interval time.Duration) {
```

Comment:
the `start` terminology makes it sound like this is non-blocking. How about calling it a loop?
```suggestion
func (m *MisbehaviorReportManager) heartbeatLoop(ctx irrecoverable.SignalerContext, interval time.Duration) {
```